apps/storybook-react/.storybook/main.ts

```diff
@@ -12,8 +12,10 @@ function getAbsolutePath(value: string): any {
 }
 const config: StorybookConfig = {
   stories: [
-    '../stories/**/*.mdx',
-    '../stories/**/*.stories.@(js|jsx|mjs|ts|tsx)',
+    '../stories/Introduction.mdx',
+    '../../../packages/design-tokens/stories/Introduction.mdx',
+    '../../../packages/design-tokens/stories/IntroductionColor.mdx',
+    '../../../packages/design-system-react/src/components/Introduction.mdx',
```

Comment:
importing specific mdx files so there is no duplication

<img width="315" alt="Screenshot 2024-11-26 at 1 36 13 PM" src="https://github.com/user-attachments/assets/dee5add3-eac0-42b5-9cac-b6a1cd05b37e">